transformer_engine/common/gemm/cublaslt_gemm.cu

```diff
@@ -205,8 +219,8 @@ void cublas_gemm(const Tensor *inputA,
                                    B,                                      /* B */
                                    Bdesc,
                                    static_cast<const void*>(&beta),        /* beta */
-                                   D,                                      /* C */
-                                   Ddesc,
+                                   nullptr,                                /* C */
```

Comment:
This breaks beta=1 case that we use for wgrad accumulation, no?

Reply:
Got it. Will address this.